app/views/view-claim.njk

```diff
@@ -34,8 +34,8 @@
         }) }}
       {% endif %}
 
-      <h1 class="govuk-caption-l govuk-!-margin-bottom-3">Claim number: {{reference}}</h1>
-      <h1 class="govuk-heading-l govuk-!-margin-bottom-7">{{ title }} <span class="govuk-tag {{ status.tagClass }}">{{ status.capitalisedtype }}</span></h1>
+      <h1 class="govuk-caption-l govuk-!-margin-bottom-3">Claim number by <a href="/view-agreement/{{applicationReference}}">{{ title }}</a> </h1>
+      <h1 class="govuk-heading-l govuk-!-margin-bottom-7"> Claim number: {{reference}} <span class="govuk-tag {{ status.tagClass }}">  {{ status.capitalisedtype }}</span></h1>
```

Comment:
Do we need the space before "  {{ status.capitalisedtype }"?

Reply:
We do but I can remove the space